File: examples/CBuildr6/ExDbEfu.cpp
//---------------------------------------------------------------------------
#include <vcl\vcl.h>
#pragma hdrstop

#include "ExDbEfu.h"
//---------------------------------------------------------------------------
#pragma link "OvcDbDLb"
#pragma link "OvcRLbl"
#pragma link "OvcDbSF"
#pragma link "OvcSF"
#pragma link "OvcEF"
#pragma link "OvcBase"
#pragma link "OvcDbPF"
#pragma link "OvcPF"
#pragma link "OvcPB"
#pragma link "OvcDbNF"
#pragma link "OvcNF"
#pragma resource "*.dfm"
TForm1 *Form1;
//---------------------------------------------------------------------------
__fastcall TForm1::TForm1(TComponent* Owner)
  : TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TForm1::BitBtn1Click(TObject *Sender)
{
  Close();    
}
//---------------------------------------------------------------------------

File: examples/Delphi/EXDBEFU.pas
{$J+} {Writable constants}

unit ExDbEFu;

interface

uses
  WinTypes, WinProcs, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, StdCtrls, Buttons, DB, DBTables,  ExtCtrls,
  OvcNF, OvcDbNF, OvcPB, OvcPF, OvcDbPF,
  OvcBase, OvcEF, OvcSF, OvcDbSF, DBCtrls, OvcRLbl, OvcDbDLb;

type
  TForm1 = class(TForm)
    Table1: TTable;
    DataSource1: TDataSource;
    OvcController1: TOvcController;
    DBNavigator1: TDBNavigator;
    BitBtn1: TBitBtn;
    Panel1: TPanel;
    OvcDbSimpleField1: TOvcDbSimpleField;
    OvcDbPictureField1: TOvcDbPictureField;
    OvcDbNumericField1: TOvcDbNumericField;
    OvcDbPictureField2: TOvcDbPictureField;
    OvcDbDisplayLabel1: TOvcDbDisplayLabel;
    OvcDbDisplayLabel2: TOvcDbDisplayLabel;
    OvcDbDisplayLabel3: TOvcDbDisplayLabel;
    OvcDbSimpleField2: TOvcDbSimpleField;
    OvcDbDisplayLabel4: TOvcDbDisplayLabel;
    OvcDbDisplayLabel5: TOvcDbDisplayLabel;
    OvcDbPictureField3: TOvcDbPictureField;
    OvcDbDisplayLabel6: TOvcDbDisplayLabel;
    procedure BitBtn1Click(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  Form1: TForm1;

implementation

{$R *.DFM}

procedure TForm1.BitBtn1Click(Sender: TObject);
begin
  Close;
end;

end.
